api/tests/Feature/Notifications/ApplicationDeadlineApproachingTest.php

```diff
@@ -22,12 +22,16 @@ class ApplicationDeadlineApproachingTest extends TestCase
 
     private ApplicationDeadlineApproaching $fixtureNotification;
 
+    private $allNotificationFamilies;
```

Comment:
Yeah, this got a little messy.  I don't think you need this $allNotificationFamilies array.  When you want the notification to send just set the enabled array to have the one family and if not, set it to empty array.  Same for all the test files.

Reply:
Yeah, that makes sense. I felt weird doing this in the first place anyway :sweat_smile: Removed in [adjust tests to be explicit about notification families](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10430/commits/50c20d16d202b2bb56981d6a762b268732139cf0)